Add I2S support on LPC55S69

mmahadevan108 opened the pull request.

mmahadevan108 committed:
drivers: i2s: Update MCUX Kconfig

Move SoC specific configs to the soc folder

Signed-off-by: Mahesh Mahadevan <mahesh.mahadevan@nxp.com>

mmahadevan108 committed:
tests: i2s: Add support for LPCXpresso55S69

Update I2S tests for LPCXpresso55S69

Signed-off-by: Mahesh Mahadevan <mahesh.mahadevan@nxp.com>

mmahadevan108 committed:
boards: lpcxpresso55s69: Add I2S support

Add I2S support for LPCXpresso55S69 board

Signed-off-by: Mahesh Mahadevan <mahesh.mahadevan@nxp.com>

MaureenHelm commented:
> I am unclear on the CI failure. The application builds successfully on my local machine

Note the failure is for the cpu1 configuration. I can reproduce it locally, although this test probably should be filtered out for this board since there is no network interface.

mmahadevan108 committed:
samples: websocket_client: Run only when netif is enabled

Add depends_on: netif to prevent running on boards that
do not have netif enabled

Signed-off-by: Mahesh Mahadevan <mahesh.mahadevan@nxp.com>

dleach02 commented on soc/arm/nxp_imx/rt6xx/Kconfig.defconfig.mimxrt685_cm33:
```diff
@@ -31,6 +31,7 @@ config SPI_MCUX_FLEXCOMM
 config I2S_MCUX_FLEXCOMM
 	default y if HAS_MCUX_FLEXCOMM
 	depends on I2S
+	select INIT_AUDIO_PLL
```
did you mean to change the rt6xx file in this LPC55S69 PR?

dleach02 replied:
Never mind. I looked at it in context of the commit and see why.
